app/main/views/admin_manager.py

```diff
@@ -63,3 +66,40 @@ def invite_admin_user():
         form=form,
         errors=errors,
     ), 200 if not errors else 400
+
+
+@main.route("/admin-users/<string:admin_user_id>/edit", methods=["GET", "POST"])
+@role_required("admin-manager")
+def edit_admin_user(admin_user_id):
+    admin_user = (data_api_client.get_user(admin_user_id))["users"]
```

Comment:
Do we need parentheses around the `data_api_client` call?

Reply:
I checked now, we don't :). I will get rid of them :D